-------------------------------------------------------------------------------
Test set: A_WarehouseTest
-------------------------------------------------------------------------------
Tests run: 19, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.138 s <<< FAILURE! - in A_WarehouseTest
tryingToTakeAProductThatDoesNotExistReturnsFalse(A_WarehouseTest)  Time elapsed: 0.022 s  <<< ERROR!
java.lang.NullPointerException
	at A_WarehouseTest.take(A_WarehouseTest.java:424)
	at A_WarehouseTest.tryingToTakeAProductThatDoesNotExistReturnsFalse(A_WarehouseTest.java:342)

methodTakeExits(A_WarehouseTest)  Time elapsed: 0.004 s  <<< FAILURE!
java.lang.AssertionError: 
NullPointerException, in call take("cheese"). 
The error was caused by the following code: Warehouse w = new Warehouse(); 
w.addProduct("coffee",2, 25);
w.addProduct("milk",3, 10);
w.take("cheese");
	at A_WarehouseTest.methodTakeExits(A_WarehouseTest.java:281)

